src/schema/resolvers/index.js

```diff
@@ -0,0 +1,15 @@
+import path from 'path'
+import fs from 'fs'
+
+const resolvers = {}
+
+fs
+  .readdirSync(__dirname)
+  .filter(file => file.indexOf('.') !== 0 && file !== 'index.js')
+  .forEach(file => {
+    const resolverPath = path.join(__dirname, file)
+    const resolver = require(resolverPath)
+    Object.assign(resolvers, resolver)
```

Comment:
My eyes hahaha

Let's use the spread operator here.

Reply:
I thought spread operator created a new object and I wanted to merge `resolver` into `resolvers`.

I'll change this line to `resolvers = {...resolvers, ...resolver};` if that's better.

Reply:
Hmm let me read.

Reply:
Yes, Nahuel. What you're doing is OK, it's similar to the method `push` of Array. But... IMHO, I think it would be more readable this way:

```javascript
resolvers = { ...resolvers, require(resolverPath))
```

There is no need for an intermediate variable here (`resolver`). And when you read this line you quickly know that you're merging these two objects. Instead, using `Object.assign` you may need to check out that method's documentation to actually know what it does.

Finally, since this iteration is very low cost, you can make it even clearer by adding a `.map` that calculates the file's path and then reduce it into a `resolvers` object:
```javascript
import path from 'path'
import fs from 'fs'

const resolvers = fs
  .readdirSync(__dirname)
  .filter(file => file.indexOf('.') !== 0 && file !== 'index.js')
  .map(file => path.join(__dirname, file))
  .reduce((acc, resolverPath) => ({ ...acc, require(resolverPath) }), {})

export default resolvers
```

Reply:
I like your solution :smile: I'll make that modification and then merge this PR. Thanks for your suggestions!